### PR TITLE
feat: Add a config to disable mentions in info messages

### PR DESCRIPTION
**Pull request by sofiedotcafe**
#1622
Added an option to disable mentions in server messages
Also the en_us.json file has placeholder names for the configs, at least for now

### CODE REVIEW

File: common/src/main/resources/assets/wynntils/lang/en_us.json [hunk 1]
```diff
@@ -75,6 +75,8 @@
   "feature.wynntils.chatMention.mentionColor.description": "What color should be used for highlighting?",
   "feature.wynntils.chatMention.mentionColor.name": "Highlight color",
   "feature.wynntils.chatMention.name": "Chat Mentions",
+  "feature.wynntils.chatMention.suppressMentionsInInfo.description": "Should there be mentions from server/info messages?",
+  "feature.wynntils.chatMention.suppressMentionsInInfo.name": "Suppress mentions in info messages",
```
Comment (kristofbolyai) [inline common/src/main/resources/assets/wynntils/lang/en_us.json]:
```suggestion
  "feature.wynntils.chatMention.suppressMentionsInInfo.description": "Should there be mentions in info messages?",
  "feature.wynntils.chatMention.suppressMentionsInInfo.name": "Suppress Mentions in Info Messages",
```

File: common/src/main/java/com/wynntils/features/chat/ChatMentionFeature.java [hunk 2]
```diff
@@ -40,6 +41,9 @@ public class ChatMentionFeature extends Feature {
     @RegisterConfig
     public final Config<String> aliases = new Config<>("");
 
+    @RegisterConfig
+    public final Config<Boolean> suppressMentionsInInfo = new Config<>(true);
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/features/chat/ChatMentionFeature.java]:
```suggestion
    public final Config<Boolean> suppressMentionsInInfo = new Config<>(false);
```

Let's not change default behavior.

Comment (sofiedotcafe) [reply common/src/main/java/com/wynntils/features/chat/ChatMentionFeature.java]: Ah, sorry

File: common/src/main/resources/assets/wynntils/lang/en_us.json [hunk 3]
```diff
@@ -75,6 +75,8 @@
   "feature.wynntils.chatMention.mentionColor.description": "What color should be used for highlighting?",
   "feature.wynntils.chatMention.mentionColor.name": "Highlight color",
   "feature.wynntils.chatMention.name": "Chat Mentions",
+  "feature.wynntils.chatMention.suppressMentionsInInfo.description": "Should there be mentions in info messages?",
```
Comment (magicus) [inline common/src/main/resources/assets/wynntils/lang/en_us.json]: This is strangely worded. Remember that a true value means that they are suppressed. Maybe "Should mentions in informational system messages be ignored?"

**Review by kristofbolyai — APPROVED** [review]

**Pull request merged**